mongodbatlas/clusters.go

```diff
@@ -30,15 +30,18 @@ type ClustersServiceOp struct {
 
 var _ ClustersService = &ClustersServiceOp{}
 
+// AutoScaling configures your cluster to automatically scale its storage and cluster tier
```

Comment:
FYI We actually have two separate autoscaling features, compute is our new one (which we'll hopefully add soon tot he provider) and diskGBEnabled, which is here, is just for storage.  In order to reduce confusion I'd change the comment to : // AutoScaling configures your cluster to automatically scale its storage

Reply:
okay, thank you ! 👍